nengo_gui/config.py

```diff
@@ -58,6 +58,9 @@ def dumps(self, uids):
                     lines.append('_viz_config[%s].has_layout=%s'
                                  % (uid, self[obj].has_layout))
 
+            elif isinstance(obj, nengo_gui.components.component.Editor):
```

Comment:
Should this be `nengo_gui.components.editor.Editor`?

Reply:
Yes, good catch.